utils.py

```diff
@@ -626,6 +626,19 @@ def get_thumbnail_icon_url_for_category(category):
     return '/subjects/%s.svg' % (icon_name.replace(' ', ''))
 
 
+def is_valid_language_code(language_code):
```

Comment:
Good idea to extract this, but you should then use it in any place that does this sort of validation in order to avoid code skew.

Reply:
I edited all the locations I think (exp, collection, topic, story, question and skill domains had used similar checks, where I made it call this function)